core/utils.py

```diff
@@ -1,23 +1,78 @@
 """Utility functions to load data from .mat .yaml and .h5 files."""
 
 import numpy as np
+from jax import numpy as jnp
 import h5py
 import os
 import yaml
 import scipy.io as spio
 import pickle
 from typing import Text
+from pynwb import NWBFile, NWBHDF5IO
+from ndx_pose import PoseEstimationSeries, PoseEstimation
 
+# Constants 
+MM_TO_M = 0.001
 
-def loadmat(filename):
+def daance_to_stac_mjx(data):
     """
-    this function should be called instead of direct spio.loadmat
-    as it cures the problem of not properly recovering python dictionaries
-    from mat files. It calls the function check keys to cure all entries
-    which are still mat-objects
+    Rearrange the data daance .mat mocap ordering
+    to internal order, and scale from millimeters to
+    meters. 
+
+    Args: mocap data arranged in dannce ordering, and
+    stored in mm.
+
+    Out: data rearranged into internal format, scaled to 
+    meters.
```

Comment:
Google style docstrings:

```suggestion
    Args:
        data: Mocap data arranged in dannce ordering, and stored in mm.

    Returns:
        Data rearranged into internal format, scaled to meters.
```

Also, please specify the actual formats in the docstring, e.g., `(frames, xyz, keypoints)` for the input.

Reply:
Fixing.

Reply:
Fixing.